addons/account_edi_ubl_cii/data/cii_22_templates.xml

```diff
@@ -328,7 +328,7 @@
                 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
                     <rdf:Description xmlns:pdfaid="http://www.aiim.org/pdfa/ns/id/" rdf:about="">
                         <pdfaid:part>3</pdfaid:part>
-                        <pdfaid:conformance>B</pdfaid:conformance>
+                        <pdfaid:conformance>A</pdfaid:conformance>
```

Comment:
I think we should add a trick in the code to update this part of the code without forcing ppl to update the module like:
```py
view = self.env.ref('account_edi_ubl_cii.account_invoice_pdfa_3_facturx_metadata').sudo()
if "<pdfaid:conformance>B</pdfaid:conformance>" in view.arch:
     view.arch = view.arch.replace("<pdfaid:conformance>B</pdfaid:conformance>", "<pdfaid:conformance>A</pdfaid:conformance>")
```
or
```py
view = self.env.ref('account_edi_ubl_cii.account_invoice_pdfa_3_facturx_metadata').sudo()
if "<pdfaid:conformance>B</pdfaid:conformance>" in view.arch:
    view.reset_arch('hard')
```
... if we don't care about customizations